blackbelt/deployment.py

```diff
@@ -5,7 +5,7 @@
 
 
 def deploy_staging():
-    branch_name = str(get_current_branch())
+    branch_name = get_current_branch()
 
     post_message("Deploying branch %s to staging" % branch_name, "#deploy-queue")
```

Comment:
I think you either need to encode here, or in post_message.

Reply:
...oh, it's going just to slack, so it's probably better to encode at the output layer. 

👍